psi4/driver/json_wrapper.py

```diff
@@ -205,13 +205,14 @@ def run_json(json_data, clean=True):
 
     # Attempt to run the computer
     try:
-        # qc_schema should be copied
-        json_data = run_json_qc_schema(copy.deepcopy(json_data), clean)
-
-    except Exception as error:
-        exc_type, exc_value, exc_traceback = sys.exc_info()
-        json_data["error"] = repr(traceback.format_exception(exc_type, exc_value,
-                                          exc_traceback))
+        # qcschema should be copied
+        json_data = run_json_qcschema(copy.deepcopy(json_data), clean)
+
+    except Exception as exc:
+        json_data["error"] = {
+            'error_type': type(exc).__name__,
+            'error_message': ''.join(traceback.format_exception(*sys.exc_info())),
+        }
         json_data["success"] = False
 
         with open(outfile, 'r') as f:
```

Comment:
While we are here could we move the output file to the scratch directory rather than `tmp` in an effort to remove some odd supercomputer bugs we have seen.

Reply:
Not following -- the only file written to here presently in scratch directory https://github.com/psi4/psi4/pull/1578/files#diff-f0c7d86da302944bdf61cc5d48e3701cR190

Reply:
Ah, ok so we are doing that. That explains an issue on my end, thanks!